### PR TITLE
fix #8423 feat(nimbus): Add codeowners

### LINKED ISSUE #8423: Add code-owners
Add @yashikakhurana  and @eliserichards as code owners, since cirrus is added, add code owners too.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3237)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- It doesn't request for you review automatically to @yashikakhurana and @eliserichards 

This commit

- Updates the codeowners file with the new user's name and also added cirrus codowners

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,3 @@
 app/tests @jrbenny35
-app/experimenter/ @jaredlockhart
+app/experimenter/ @jaredlockhart @yashikakhurana @eliserichards
+app/cirrus/ @jaredlockhart @yashikakhurana
```
Comment (jaredlockhart) [inline .github/CODEOWNERS:3]: No @jeddai ?

Comment (yashikakhurana) [reply .github/CODEOWNERS:3]: oops sorry forgot

Comment (yashikakhurana) [reply .github/CODEOWNERS]: done!

**Review by eliserichards — APPROVED** [review]
Perfect!

**Comment by jeddai** [comment]
I know this has already merged but this is a cool github feature I am only just now learning about!

**Comment by yashikakhurana** [comment]
@jeddai I also came to know when I was refactoring app to experimenter :D

**Pull request merged**